UKHomeOffice/cerberus-service
# COP-10033: Add tests to verify Highest threat level on task list and task details page

## Description
Add tests to verify Highest threat level on task list and task details page

## To Test
npm run cypress:runner
run the following tests
`task-management.spec.js`
`Should check rule matches details on task management page`
`Should check selector matches details on task management page`

`task-version-details.spec.js`
`Should verify single task created for the same target with different versions with different passengers information`
`Should verify task Display highest threat level in task details`

`task-details.spec.js`
`Vehicle and Trailer - Unknown values in Task List & Task Summary`
`Vehicle - Unknown values in Task List and Task summary`
`Should display correct values for PERFDRIVER Driver, Booking, Vehicle, tasklist, task summary`

`create-tasks.spec.js`
`Should create a task with payload contains risks array as null`

## Developer Checklist

\* Required
